compose/cli/command.py

```diff
@@ -115,9 +116,21 @@ def unicode_paths(paths):
     return None
 
 
+def get_profiles_from_options(options, environment):
+    profile_option = options.get('--profile')
+    if profile_option:
+        return profile_option
+
+    profiles = environment.get('COMPOSE_PROFILE')
+    if profiles:
+        return profiles.split(':')
```

Comment:
`:` as a separator is hard coded here
* should it be something else?
* should it be configurable?

Reply:
Curious why wouldn't it be simply `,`?

Reply:
No specific reason for `:` here, I just oriented on the separator in [`COMPOSE_FILE`](https://docs.docker.com/compose/reference/envvars/#compose_file) (at least on Linux/macOS) but it could be `,`, whitespace or any of it equally well.

The [spec only defines the regex `[a-zA-Z0-9][a-zA-Z0-9_.-]+`](https://github.com/compose-spec/compose-spec/commit/36e14e20259425605016cabec75a6a28606d8dc8#diff-bc6661da34ecae62fbe724bb93fd69b91a7f81143f2683a81163231de7e3b545R1525) for valid profile names. So anything else could be used as separator.

Reply:
I would prefer `,` as separator, to make it similar to a list in python.

Reply:
Ok, then: I updated the PR to use a hard coded `,` as delimiter here